tests/e2e/temporal_selection/temporal_selection.spec.js

```diff
@@ -0,0 +1,186 @@
+        // Mock current date
+        const currentDate = moment().endOf('day')
+        const mockCurrentDate = currentDate.format('MM-DD HH:mm:ss')
+
+        await page.getByRole('checkbox', { name: 'Use a recurring date range' }).check()
+
+        await page.getByRole('button', {
+          name: 'Apply',
+          exact: true
+        }).click()
+
+        // Check that start date is automatically applied if one isn't provided
+        await expect(page.getByText('01-01 00:00:00')).toBeVisible()
+
+        // Check that the end date was defaulted to current day
+        await expect(page.getByText(mockCurrentDate)).toBeVisible()
+
+        // Check that range was applied
+        await expect(page.getByText('1960 - 2025')).toBeVisible()
+      })
+
+      test('applies values for start and end when they have been selected', async ({ page }) => {
+        // Select a start date
+        await page.getByRole('textbox', { name: 'Start Date' }).click()
+        await page.getByRole('cell', { name: '2021' }).click()
+        await page.getByRole('cell', { name: 'Mar' }).click()
+        await page.getByRole('cell', { name: '5' }).first().click()
+
+        // Select an end date
+        await page.getByRole('textbox', { name: 'End Date' }).click()
+        await page.getByRole('cell', { name: '2025' }).click()
+        await page.getByRole('cell', { name: 'Jul' }).click()
+        await page.getByRole('cell', { name: '10' }).click()
+
+        await page.getByRole('checkbox', { name: 'Use a recurring date range' }).check()
+        await page.getByRole('button', {
+          name: 'Apply',
+          exact: true
+        }).click()
+
+        // Check that all selections have been applied
+        await expect(page.getByText('03-05 00:00:00')).toBeVisible
+        await expect(page.getByText('07-10 23:59:59')).toBeVisible
+        await expect(page.getByText('2021 - 2025')).toBeVisible
+      })
+    })
+  })
+
+  test.describe('when temporal search results are recieved from edsc store', () => {
+    test.beforeEach(async ({ page }) => {
+      await page.goto('/search?qt=2021-03-04T00%3A00%3A00.000Z%2C2022-04-02T23%3A59%3A59.999Z')
+      await expect(page.getByTestId('collection-results-list')).toBeVisible()
+      await page.getByRole('button', { name: 'Open temporal filters' }).click()
+    })
+
+    test('search parameters are used to render dropdown menu', async ({ page }) => {
+      // Check if the Start Date input is blank
+      const startDateInput = page.getByRole('textbox', { name: 'Start Date' })
+      await expect(startDateInput).toHaveValue('2021-03-04 00:00:00')
+
+      // Check if the End Date input is blank
+      const endDateInput = page.getByRole('textbox', { name: 'End Date' })
+      await expect(endDateInput).toHaveValue('2022-04-02 23:59:59')
+    })
+
+    test('clicking Clear from DatePicker modal clears the date', async ({ page }) => {
```

Comment:
Because there are two functions that handle the clear button within the datepicker (setStartDate and setEndDate) make two different tests that start with both fields populated and then only press one clear button. Then show only the correct field is empty and the other is still populated